### ISSUE TITLE
Notcurses hangs when processing the DA1 query on the rxvt-unicode-sixel fork

### ISSUE THREAD

**Issue by j4james** [dankamongmen/notcurses#2255]
* LANG="C.UTF-8"
* TERM="rxvt-unicode"
* notcurses version 2.4.5 (commit f5ec1406fea2f6fd446fe5714b8cae4ca2e708d8)
* Terminal https://github.com/saitoha/rxvt-unicode-sixel

Steps to reproduce:
1. Run `notcurses-info`
2. It hangs

What seems to be the problem is that the sixel fork is responding to the DA1 query with `\e[?1;2;4c`, which the new input parser doesn't like. I don't know if this is the right solution, but I could get it to work again by adding `{ "[?1;2;4c", da1_cb, },` to the `csis` table in `build_cflow_automaton` after the `{ "[?1;2c", da1_cb, },` entry.

**Comment by dankamongmen** [comment]
having some problems getting a working rxvt-unicode-sixel binary built, first off...

**Comment by dankamongmen** [comment]
your solution sounds perfectly workable, and probably the ideal one. would you like to submit a PR to pick up the stat? if not, i'll go ahead and commit that fix or something very much like it, crediting you.

thanks for bringing this to my attention! i might look for a more generic catch-all DA1, perhaps in addition to this change.

**Comment by dankamongmen** [comment]
> Bottom line is I wouldn't try and read too much in that sequence, since you can already detect their sixel capability via the `XTSMGAPHICS` reports.

yep, but it becomes more important with #2257, since we now require that DA1 advertise sixel.

i'm pretty sure the fix i committed for #2257 actually gets this handled, as well. if you could test it out, i'd appreciate it. `\e[1;2;4c` isn't going to match either of the two remaining specific paths (`\e[6c`, `\e[7c`), so it's going to hit `da1_attrs_cb` with both the `2` and the `4` interpreted as features, which ought be sufficient unless i have some horrible bug in my DFA generator.
